tests/behavior/s3.rs

```diff
@@ -29,5 +31,11 @@ async fn behavior() -> Result<()> {
         return Ok(());
     }
 
-    BehaviorTest::new(Operator::new(acc.unwrap())).run().await
+    jaeger_tracing(
```

Comment:
A bit complex to me.

How about:

```rust
if let Ok(addr) = env::var("OPENDAL_JAEGER_ADDR") {
   // Init jaeger here.
}

// Test logic.

if let Ok(addr) = env::var("OPENDAL_JAEGER_ADDR") {
   // Report to jaeger here.
}
```

Reply:
Anyway, the jaeger test in behavior test is not a must-have part. This PR is good enough to get merged (Changes under `services/*` and `minitrace` in `Cargo.toml`). Please consider splitting them into different PRs.